docs/getting_started/self_host.md

```diff
@@ -0,0 +1,243 @@
+# Self-hosting
+
+In this tutorial, you will learn how to self- host Windmill. 
+We will use [Traefik][traefik] as reverse proxy.  
+
+At the end of this tutorial, you will have a full-fledged version of Windmill,
+with LSP and a Postgres database running on your own server. We will deploy 
+with automatic LetsEncrypt encryption, handled by Traefik, using sensible
+defaults. 
+
+All the files required to achieve that are provided at the end of this
+tutorial in the [Example files](#example-files) section. These files are a
+good **starting point** for a internet-facing deployment of Windmill,
+and should be used as such. Without configuration they will not work and
+are not suitable for production deployments.
+
+## Intro
+
+To follow this tutorial, you will need Docker, `docker-compose` and a clone
+of the main [Windmill repository][windmill]:  
+
+`$ git clone git@github.com:windmill-labs/windmill.git`
+
+
```

Comment:
I would not clone the repo or build the containers from scratch. They're all available from ghcr directly.